jcl/src/java.base/share/classes/openj9/tools/attach/diagnostics/base/DiagnosticProperties.java

```diff
@@ -374,9 +378,10 @@ public static Properties makeExceptionProperties(Exception e) {
 		props.put(IPC.PROPERTY_DIAGNOSTICS_ERROR, Boolean.toString(true));
 		props.put(IPC.PROPERTY_DIAGNOSTICS_ERRORTYPE, e.getClass().getName());
 		String msg = e.getMessage();
-		if (null != msg) {
-			props.put(IPC.PROPERTY_DIAGNOSTICS_ERRORMSG, msg);
+		if (null == msg) {
+			msg = "Error at " + e.getStackTrace()[0].toString(); //$NON-NLS-1$
```

Comment:
I think `toString()` here is not required. Also I would suggest `String.format()` over String concatenation though here is probably a slow path anyway.

Reply:
There is an implicit `toString()`, but I wanted to make it explicit.
I like the String.format() idea.  Done.